examples/multiplayer-lan/rust/src/bullet.rs

```diff
@@ -0,0 +1,60 @@
+use godot::classes::{CharacterBody2D, ICharacterBody2D, ProjectSettings};
+use godot::prelude::*;
+
+use crate::NetworkId;
+
+const SPEED: f32 = 500.0;
+const LIFETIME: f64 = 2.0;
+
+#[derive(GodotClass)]
+#[class(base=CharacterBody2D)]
+pub struct Bullet {
+    gravity: f64,
+    direction: Vector2,
+    // who shot the bullet
+    #[var]
+    pub network_id: NetworkId,
+    // dont want the bullets to live forever
+    time_left: f64,
+    base: Base<CharacterBody2D>,
+}
+
+#[godot_api]
+impl ICharacterBody2D for Bullet {
+    fn init(base: Base<CharacterBody2D>) -> Self {
+        let gravity: f64 = Result::expect(
+            ProjectSettings::singleton()
+                .get_setting("physics/2d/default_gravity".into())
+                .try_to::<f64>(),
+            "default setting in Godot",
```

Comment:
…what does it error message do? What does it inform user about? "default setting in godot" informing about… not being able to cast given setting to float?